tidb-cloud/tidb-cloud-console-auditing.md

```diff
@@ -1,122 +1,170 @@
-| AuditEventCreateGCPVpcPeering | Create a GCP VPC Peering | 
-| AuditEventListAWSVpcPeering | List all AWS VPC Peerings in a project |
-| AuditEventListGCPVpcPeering | List all GCP VPC Peerings in a project |
-| AuditEventDeleteAWSVpcPeering | Delete an AWS VPC Peering |
-| AuditEventDeleteGCPVpcPeering | Delete a GCP VPC Peering |
-| AuditEventGetProjectTrafficFilter | Get traffic filter list of a project |
-| AuditEventUpdateProjectTrafficFilter | Update traffic filter list of a project |
-| AuditEventGetTrafficFilter | Get traffic filter list of a cluster |
-| AuditEventUpdateTrafficFilter | Update traffic filter list of a cluster |
-| AuditEventCreateProjectCIDR | Create a new project CIDR |
-| AuditEventGetProjectCIDR | List the CIDR of a region |
-| AuditEventGetProjectRegionCIDR | List all CIDRs in a project |
-| AuditEventDeleteBackupInRecycleBin | Delete backups of deleted clusters in recycle bin |
-| AuditEventChangeClusterRootPassword | Reset the root password of a cluster |
-| AuditEventCreateImportTask | Create an import task |
-| AuditEventCancleImportTask | Cancel an import task |
-| AuditEventExitImportTask | Exit an import task |
-| AuditEventCreateCluster | Create a cluster |
-| AuditEventDeleteCluster | Delete a cluster |
-| AuditEventScaleCluster | Scale a cluster |
-| AuditEventCreateBackup | Make a backup |
-| AuditEventDeleteBackup | Delete a backup |
-| AuditEventRestoreBackup | Restore from a backup |
-| AuditEventUpdateAuditLogStatus | Enable or disable database audit logging |
-| AuditEventCreateAuditLogAccessRecord | Add database audit log filter conditions |
-| AuditEventDeleteAuditLogAccessRecord | Delete database audit log filter conditions |
-| AuditEventUpdateUserRole | Modify user roles |
-
-## Audit log storage policy
-
-- The audit log information is stored in AWS ES.
-- The storage time is 90 days, after which the audit logs will be automatically cleaned up.
-
-## View audit logs
-
-The console audit logs are temporarily accessible only to internal personnel of TiDB Cloud. If you need to view the logs, contact [PingCAP support team](/tidb-cloud/tidb-cloud-support.md).
-
-## Audit log fields
-
-The fields in audit logs include basic fields and extended fields.
-
-The basic fields are as follows:
+To disable console audit logging, take the following steps:
 
-| Field name | Data type | Description |
-|---|---|---|
-| timestamp | timestamp | Time of event |
-| auditEventType | string | Event type |
-| userID | uint64 | User ID |
-| clientIP | string | Client IP |
-| isSuccess | bool | Event result |
+1. In the upper-right corner of the [TiDB Cloud console](https://tidbcloud.com/), click <MDSvgIcon name="icon-top-organization" /> **Organization** > **Console Audit Logging**.
+2. Click **Setting** in the upper-right corner, and then disable console audit logging.
+
+## View console audit logs
+
+You can only view the console audit logs of your organization. 
 
```

Comment:
```suggestion

> **Note:**
>
> - If it is the first time your organization enables console audit logging, the console audit logs are empty. You can click refresh to get the latest logs.
> - If it has been more than 90 days since console audit logging was disabled, the console audit logs are empty.
```